docs/tutorials/simple-date-clock-application.adoc

```diff
@@ -0,0 +1,482 @@
+For a local deployment, you can use minikube, which is basically https://minikube.sigs.k8s.io/docs/start/[a small Kubernete cluster on your machine].
+
+[IMPORTANT]
+The setup of a Kubernete cluster is not mandatory to use CloudHarness.
+This step is proposed in this tutorial to show you how to deploy an application generated with CloudHarness on a local cluster and to appreciate how much CloudHarness reduces the pain of writing deployment configuration artifacts.
+
+First, we will create a minikube cluster with 6Gb of memory, 4 CPUs and a disk of more or less 2GB.
+
+.Creating minikube cluster
+[source,bash]
+----
+minikube start --memory="6000mb" --cpus=4
+----
+
+To check if the creation of the cluster went right, run `kubectl cluster-info`.
+
+The setup of the cluster is not yet entirely finished.
+To conclude this, you need to enable an addon.
+Kubernete comes with various addon to deal with various aspects of your cluster.
+As the app we will develop/run are sometimes webapps, they need to be exposed "outside" of your cluster.
+The https://kubernetes.io/docs/concepts/services-networking/ingress/[`ingress` addon] helps in exposing HTTP/HTTPS routes from outside your cluster to services inside your cluster.
+Its activation is fairly simple using minikube.
+
+.Activating `ingress` addon
+[source,bash]
+----
+minikube addons enable ingress
+----
+
+Finally, it's important to activate some environement variables *in each shell you'll use to run commands*.
+Those variable will connect the different tools (especially skaffold) to use the docker environement inside your minikube cluster to build your apps images.
+
+.Activating the different environement variables
+[source,bash]
+----
+minikube docker-env | source
+# OR
+eval $(minikube docker-env)
+----
+
+
+== Run the projects and examples from the cloud-harness repository
+
+To be able to build, locally deploy and run the existing app (or any app you'll develop), you have to generate a specific `helm` configuration that will be use by skaffold to build/deploy/run your apps.
+The generation of those `helm` artifacts is done using `harness-deployment` with specific options.
+The next snippet show how to generate the `helm` configuration, disabling TLS configuration, enabling the local environement, selecting the `azathoth` namespace to deploy the app, and run it for the `azathoth.local` domain (you can use whatever domain name you want).
+The namespace can be changed depending on in which namespace you want to deploy your app in your cluster.
+This configuration is generated for *all* the app present in the `applications` folder in the `cloud-harness` repository.
+
+Depending on the namespace you use for the generation of your configuration, you need to create it yourself.
+Create the namespace first, then generate the deployment configuration.
+
+.Creating the namespace for your minikube cluster
```

Comment:
I would suggest to run:
```
kubectl config use-context minikube
```
to avoid having the user running the commands in a different context (f.e. a previously configured remote cluster)

Reply:
This commands replace the need for `minikube docker-env | source` in every shell?
I'll add few lines to deal with it :)

Reply:
I'll add that :) thanks a lot for the suggestion

Reply:
No. The one I mentioned changes the **kubectl** context globally, the one you do changes the **docker** 'context' in the shell you are.

In other words the kubectl command let's you define if you want to reach cluster A or cluster B (A and B can be our production clusters or your local minikube for example).

The docker command let's you define if you want the docker images to be built inside minikube or in your system

Reply:
I added one line about it with a link toward the documentation. Thanks for the explanation, it's way more clear to me :)